test/integration/targets/aws_elasticbeanstalk_app/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for aws_elasticbeanstalk_app
```

Comment:
If possible, could you use "{{ resource_id }}" that is available here for the name? It makes it easier to identify what tests are creating for CI.

Reply:
I don't see a reference to this in any other test? Do you have an example?

Reply:
ec2_group is an example, though I think all AWS integration tests are using it in some capacity. https://github.com/ansible/ansible/blob/devel/test/integration/targets/ec2_group/

A benefit of using the resource_prefix var is that multiple tests can be run in CI at once, whereas if you name your resources something static tests might interfere with each other.